### ISSUE TITLE
OverlayClusters multiprocessor does not return to intepreter with matplotlib > 2.1

### ISSUE THREAD

**Issue by kmdouglass** [kmdouglass/bstore#91]
After closing the plot window from the OverlayClusters multiprocessor (or after finishing filtering clusters), the matplotlib event loop continues running. This means that the loop has to be forced to stop (pressing the "Stop" button in a Jupyter notebook or Ctrl-C'ing in IPython) to continue working.

This bug is new as of Matplotlib 2.1 or 2.2.

This might be related to this new deprecation warning that appears when running the multiprocessor:

```
/home/douglass/src/bstore/bstore/multiprocessors.py:525: MatplotlibDeprecationWarning: The start_event_loop_default function was deprecated in version 2.1.
  self._fig.canvas.start_event_loop_default()
```

**Comment by kmdouglass** [comment]
This is fixed by changing `start_event_loop_default()` to `start_event_loop()`.

https://matplotlib.org/api/backend_bases_api.html?highlight=start_event_loop_default#matplotlib.backend_bases.FigureCanvasBase.start_event_loop

**Comment by kmdouglass** [comment]
Fixed by #92
